core/src/main/java/com/github/kaktushose/jda/commands/dispatching/events/ModalReplyableEvent.java

```
@@ -71,9 +72,7 @@ public void replyModal(String modal, Function<ModalBuilder, ModalBuilder> callba
             log.debug("Replying to interaction \"{}\" with Modal: \"{}\". [Runtime={}]", definition.displayName(), modalDefinition.displayName(), runtimeId());
             modalCallback.replyModal(builtModal).queue();
         } else {
-            throw new IllegalArgumentException(
-                    String.format("Cannot reply to '%s'! Please report this error to the jda-commands devs!", event.getClass().getName())
-            );
+            throw new JDACException.Internal("Cannot reply to '%s'! Please report this error to the jda-commands devs!", event.getClass().getName());
```

"Please report this error to the devs of jda-commands." can be removed, would be duplicated